lib/core/test/unit/Cardano/WalletSpec.hs

```diff
@@ -129,12 +130,15 @@ import Test.Hspec
 import Test.QuickCheck
     ( Arbitrary (..)
     , Property
+    , arbitraryBoundedEnum
     , choose
     , elements
     , property
     , withMaxSuccess
     , (==>)
     )
+import Test.QuickCheck.Instances.Time
+    ()
```

Comment:
I would recommend replacing this with `Test.Utils.Time.genUniformTime` to get a wider range of `UTCTime` values. The instance supplied by the package above generates times that are clustered around 1864:

See: https://github.com/input-output-hk/cardano-wallet/wiki/Blackboard#investigate-the-arbitrary-instance-for-utctime-in-testquickcheckinstancestime